.gitignore

```diff
@@ -1 +1,3 @@
-vendor/
\ No newline at end of file
+vendor/
+kura
+.DS_Store
```

Comment:
Aside, you can add the following to ~/.gitconfig:
```
[core]
  excludesfile = ~/.gitignore-global
```

And add .DS_Store and the like there instead.